includes/class-edd-cli.php

```diff
@@ -363,19 +363,39 @@ public function customers( $args, $assoc_args ) {
 	 *
 	 * ## OPTIONS
 	 *
-	 *
+	 * --email=<customer_email>: The email address of the customer to retrieve 
+	 * 
 	 * ## EXAMPLES
 	 *
-	 * wp edd sales
+	 * wp edd sales 
+	 * wp edd sales --email=john@test.com
 	 */
 	public function sales( $args, $assoc_args ) {
+		
+		$email = isset( $assoc_args ) && array_key_exists( 'email', $assoc_args ) ? $assoc_args['email'] : false;
 
 		$sales = $this->api->get_recent_sales();
 
 		if( empty( $sales ) ) {
 			WP_CLI::error( __( 'No sales found', 'easy-digital-downloads' ) );
 			return;
 		}
+		
+		if( !empty($email)) {
```

Comment:
By doing it after the fact, it's possible that you might not get the correct number of items back since you are filtering after the query has been run. The `get_recent_sales()` method in the API Looks for an email address in the `$wp_query->query_vars['email']` values, so it'd be better to simply do something like this before the line that calls `get_recent_sales()`

```
global $wp_query;
$wp_query->query_vars['email'] = $email;
```
Can you give that a shot instead of filtering after?